clusters/build-clusters/01_cluster/openshift/api_audit_log/vector-audit-log_daemonset.yaml

```diff
@@ -22,7 +22,7 @@ spec:
       dnsPolicy: Default
       containers:
       - name: vector-audit-log
-        image: docker.io/timberio/vector:0.9.0-alpine
+        image: docker.io/timberio/vector:0.11.1-alpine
```

Comment:
What is the reason we have multiple manifests for this?

Reply:
No idea. I think @hongkailiu added those manifests. Shall we put them all in one file?